src/Elevator.java

```diff
@@ -1,4 +1,44 @@
+import java.lang.Math;
+import java.time.LocalTime;
+import java.util.ArrayList;
 
-public class Elevator {
+public class Elevator implements Runnable {
+  private int currentFloor;
+  private boolean isStationary;
+  private Middleman middleman;
+  private Direction direction;
+  private ArrayList<ElevatorButton> buttons;
 
+  public Elevator(Middleman middleman) {
+    this.middleman = middleman;
+    isStationary = true;
+    currentFloor = 0;
+    direction = Direction.UP;
+    buttons = new ArrayList<ElevatorButton>();
+
+    //Create an array of elevator buttons
+    for(int i = 0;i < 10;i++) {
+      buttons.add(new ElevatorButton());
```

Comment:
why are there 10 elevator buttons? Elevator constructor should probably take in a "numFloors" variable to know how many to create